style.css

```diff
@@ -334,3 +334,77 @@ button {
   line-height: 20px;
   color: #253858;
 }
+
+.contact-form {
+  background-image: url(src/Contact-me.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contactContainer {
+  background-image: url(src/contact-shapes-mobile.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: right;
+}
+
+.form-title {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  color: #fff;
+  text-align: center;
+}
+
+.content {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  color: #ebebff;
+  text-align: center;
+  margin: 10px;
+}
+
+#name,
+#email {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  box-sizing: border-box;
+  padding: 12px;
+  margin-top: 10px;
+  margin-bottom: 8px;
+}
+
+#message {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  background-color: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  border: 2px solid #ccc;
+  border-radius: 8px;
+  box-sizing: border-box;
+  width: 95%;
+  height: 150px;
+}
```

Comment:
- [ ] Kindly you can make the `width: 95%;` smaller like just `90%` to be like the Figma design.
- [ ] And maybe you can add some `margin-top` and `margin-bottom` to be like the Figma design.

![image](https://user-images.githubusercontent.com/35535130/155251168-19f98bec-d216-4040-9037-6d09f7767073.png)